### PR TITLE
convert time "create time" from timestamp to long

### PR DESCRIPTION
**Pull request by shajyhia**

### CODE REVIEW

**Comment by oritstone** [comment]
@shajyhia please describe the bug you are trying to fix - what is the functional problem

File: engine/data/score-data-api/src/main/resources/META-INF/database/score.changes.xml [hunk 1]
```diff
@@ -280,4 +280,22 @@ http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbch
         </createTable>
     </changeSet>
 
+    <!--<changeSet id="alter OO_EXECUTION_QUEUES CREATE_TIME" author="engine">-->
```
Comment (oritstone) [inline engine/data/score-data-api/src/main/resources/META-INF/database/score.changes.xml]: Why did you add this commented block?

**Comment by shajyhia** [comment]
convert the property create_time of execution_queue table from type timestamp to long in order to support timezone

File: engine/queue/score-queue-impl/src/main/java/io/cloudslang/engine/queue/repositories/ExecutionQueueRepository.java [hunk 2]
```diff
@@ -26,7 +25,7 @@
  */
 public interface ExecutionQueueRepository {
 
-    List<ExecutionMessage> poll(Date createDate, String workerId, int maxSize, ExecStatus... statuses);
+    List<ExecutionMessage> poll_(String workerId, int maxSize, ExecStatus... statuses);
```
Comment (oritstone) [inline engine/queue/score-queue-impl/src/main/java/io/cloudslang/engine/queue/repositories/ExecutionQueueRepository.java]: remove _

File: engine/queue/score-queue-impl/src/main/java/io/cloudslang/engine/queue/repositories/ExecutionQueueRepositoryImpl.java [hunk 3]
```diff
@@ -283,7 +283,7 @@ public int getBatchSize() {
 
 
 	@Override
-	public List<ExecutionMessage> poll(Date createTime, String workerId, int maxSize, ExecStatus... statuses) {
+	public List<ExecutionMessage> poll_(String workerId, int maxSize, ExecStatus... statuses) {
```
Comment (oritstone) [inline engine/queue/score-queue-impl/src/main/java/io/cloudslang/engine/queue/repositories/ExecutionQueueRepositoryImpl.java]: use poll()

File: engine/queue/score-queue-impl/src/main/java/io/cloudslang/engine/queue/services/ExecutionQueueServiceImpl.java [hunk 4]
```diff
@@ -120,8 +119,8 @@ public void enqueue(List<ExecutionMessage> messages) {
 
 	@Override
 	@Transactional(readOnly = true)
-	public List<ExecutionMessage> poll(Date createDate, String workerId, int maxSize, ExecStatus... statuses) {
-		List<ExecutionMessage> result = executionQueueRepository.poll(createDate, workerId, maxSize, statuses);
+	public List<ExecutionMessage> poll(String workerId, int maxSize, ExecStatus... statuses) {
+		List<ExecutionMessage> result = executionQueueRepository.poll_(workerId, maxSize, statuses);
```
Comment (oritstone) [inline engine/queue/score-queue-impl/src/main/java/io/cloudslang/engine/queue/services/ExecutionQueueServiceImpl.java]: use poll()

File: engine/queue/score-queue-impl/src/main/java/io/cloudslang/engine/queue/repositories/ExecutionQueueRepository.java [hunk 5]
```diff
@@ -26,7 +25,7 @@
...
 
 	List<ExecutionMessage> poll(String workerId, int maxSize, ExecStatus... statuses);
```
Comment (oritstone) [inline engine/queue/score-queue-impl/src/main/java/io/cloudslang/engine/queue/repositories/ExecutionQueueRepository.java]: method name should change to pollRecovery()

File: engine/queue/score-queue-impl/src/main/java/io/cloudslang/engine/queue/services/ExecutionQueueServiceImpl.java [hunk 6]
```diff
@@ -133,7 +132,7 @@ public void enqueue(List<ExecutionMessage> messages) {
 
 	@Override
 	@Transactional(readOnly = true)
-	public List<ExecutionMessage> poll(String workerId, int maxSize, ExecStatus... statuses) {
+	public List<ExecutionMessage> pollRecovery(String workerId, int maxSize, ExecStatus... statuses) {
 		return executionQueueRepository.poll(workerId, maxSize, statuses);
```
Comment (oritstone) [inline engine/queue/score-queue-impl/src/main/java/io/cloudslang/engine/queue/services/ExecutionQueueServiceImpl.java]: use pollRecovery()

File: worker/worker-manager/score-worker-manager-impl/src/main/java/io/cloudslang/worker/management/services/InBuffer.java [hunk 7]
```diff
@@ -94,12 +94,12 @@ private void init(){
...
-            pollCounter = pollCounter + 1;
+//            pollCounter = pollCounter + 1;
             // we reset the currentCreateDate every 100 queries , for the theoretical problem of records
             // with wrong order of create_time in the queue table.
-            if ((pollCounter % 100) == 0) {
-                currentCreateDate = new Date(0);
-            }
+//            if ((pollCounter % 100) == 0) {
+//                currentCreateDate = new Date(0);
+//            }
```
Comment (oritstone) [inline worker/worker-manager/score-worker-manager-impl/src/main/java/io/cloudslang/worker/management/services/InBuffer.java]: remove comments

File: worker/worker-manager/score-worker-manager-impl/src/main/java/io/cloudslang/worker/management/services/InBuffer.java [hunk 8]
```diff
@@ -126,7 +126,7 @@ private void fillBufferPeriodically() {
 
                         if (!newMessages.isEmpty()) {
                             // update currentCreateDate;
-                            currentCreateDate = new Date(newMessages.get(newMessages.size()-1).getCreateDate().getTime() - 100);
+//                            currentCreateDate = new Date(newMessages.get(newMessages.size()-1).getCreateDate() - 100);
```
Comment (oritstone) [inline worker/worker-manager/score-worker-manager-impl/src/main/java/io/cloudslang/worker/management/services/InBuffer.java]: remove comments + unused param currentCreateDate

**Comment by orius123** [comment]
Why do you need poll and pollRecovery?
